jenkinsfile-runner-steward-image/scripts/run.sh

```diff
@@ -45,7 +45,7 @@ function main() {
   check_required_env_vars "${PARAM_VARS_MANDATORY[@]}"
 
   echo "Cloning pipeline repository $PIPELINE_GIT_URL"
-  with_termination_log git clone "$PIPELINE_GIT_URL" .
+  with_termination_log with_retries 5 5 git clone "$PIPELINE_GIT_URL" .
```

Comment:
Setting both the retry interval and retry timeout to 5 seconds doesn't make much sense.

Reply:
Sorry, my bad. I forgot to change it